test/e2e/setup.go

```diff
@@ -196,6 +267,15 @@ func setup(ctx context.Context) error {
 		return err
 	}
 
+	log.Infof("Starting Selenium Grid")
```

Comment:
How costly are the UI tests? I would love to see them hidden behind a flag, so we can run them when required.

Reply:
In terms of resources or time? Resources wise I've had to run the portal as part of the E2E suite. And time wise they don't take any longer than a minute or so. But we can still hide them behind a flag that defaults to on if need be